javascript/javascript3/week2/exercise 3/part 2/Part2.js

```diff
@@ -0,0 +1,25 @@
+let currentLocation = document.getElementById("location");
+document.body.appendChild(currentLocation);
+
+function getLocation() {
+    return new Promise((resolve, reject) => {
+        if (navigator.geolocation) {
+            resolve();
+        } else {
+            reject();
+        }
+    })
+}
+getLocation()
+    .then(() => {
+        navigator.geolocation.getCurrentPosition(pos => {
+            let lat = pos.coords.latitude;
+            let long = pos.coords.longitude;
+            currentLocation.innerHTML = `Latitude; ${lat} 
+           <br>
+           Longitude; ${long}`
+        })
```

Comment:
this logic, calling the `getCurrentPosition` function, should be inside the Promise at line 5-11 :)

this exercise looks very much like the answer from https://github.com/27ManishaPatel/hyf-homework/blob/3329c2b74ccb963901ded8c2c4b34310f3780bfb/javascript/javascript3/week2/warmup/script.js#L79

did you work together?

Reply:
no,i did my homeworks with my son .
generally before i startet to do homeworks i search about them in google and find videos which help me to did them and sometimes I asked my son ,i dont know how they looks likes each other

Reply:
ok great :)